pr_insight/tools/ticket_pr_compliance_check.py

```
@@ -1,94 +1,126 @@
-
-                except Exception as e:
-                    get_logger().error(
-                        f"Error getting issue {original_issue_number}: {e}",
-                        artifact={"traceback": traceback.format_exc()}
-                    )
-    except Exception as e:
-        get_logger().error(
-            f"Error extracting tickets: {e}",
-            artifact={"traceback": traceback.format_exc()}
-        )
+            tickets = extract_ticket_links_from_pr_description(user_description, git_provider.repo, git_provider.base_url_html)
```

Ruff

Line too long (127 > 120)

(E501)